ci-operator/step-registry/olmv1/performance/olmv1-performance-commands.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+# cat /etc/os-release
+# oc config view
+# oc projects
+oc version
+oc get co
+oc get nodes
+# python --version
+pushd /tmp
+# python -m virtualenv ./venv_qe
+# source ./venv_qe/bin/activate
+
+# connected to ES server
+ES_SECRETS_PATH=${ES_SECRETS_PATH:-/secret}
+ES_HOST=${ES_HOST:-"search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"}
+ES_PASSWORD=$(cat "${ES_SECRETS_PATH}/password")
+ES_USERNAME=$(cat "${ES_SECRETS_PATH}/username")
+if [ -e "${ES_SECRETS_PATH}/host" ]; then
+    ES_HOST=$(cat "${ES_SECRETS_PATH}/host")
+fi
+
+LOG_LEVEL=${LOG_LEVEL:-debug}
+KUBE_DIR=${KUBE_DIR:-/tmp}
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.6.8}
+PERFORMANCE_PROFILE=${PERFORMANCE_PROFILE:-default}
+CHURN=${CHURN:-true}
+PPROF=${PPROF:-true}
+ARCHIVE=${ARCHIVE:-true}
+WORKLOAD=${WORKLOAD:-olm}
+QPS=${QPS:-20}
+BURST=${BURST:-20}
+GC=${GC:-true}
+EXTRA_FLAGS=${EXTRA_FLAGS:-}
+UUID=${UUID:-$(uuidgen)}
+ITERATIONS=${ITERATIONS:-30}
+AIRFLOW_CTX_DAG_ID=${AIRFLOW_CTX_DAG_ID:-}
+PROW_JOB_ID=${PROW_JOB_ID:-}
+BUILD_ID=${BUILD_ID:-}
+ADDITIONAL_PARAMS=${ADDITIONAL_PARAMS:-}
+ES_SERVER=${ES_SERVER:-"https://$ES_USERNAME:$ES_PASSWORD@$ES_HOST"}
+METRICS_ENDPOINT=${METRICS_ENDPOINT:-metrics-endpoint.yml}
+
+cp /go/src/github.com/openshift/openshift-tests-private/test/extended/operators/benchmark/metrics-endpoint.yml ${KUBE_DIR}
+
+KUBE_BURNER_URL="https://github.com/kube-burner/kube-burner-ocp/releases/download/v${KUBE_BURNER_VERSION}/kube-burner-ocp-V${KUBE_BURNER_VERSION}-linux-x86_64.tar.gz"
+curl --fail --retry 8 --retry-all-errors -sS -L "${KUBE_BURNER_URL}" | tar -xzC "${KUBE_DIR}/" kube-burner-ocp
+
+# git clone https://github.com/kube-burner/kube-burner-ocp.git --branch main --depth 1
+# pushd kube-burner-ocp
+# make build
+# ./bin/amd64/kube-burner-ocp olm -h
+
+# REPO_URL="https://github.com/cloud-bulldozer/e2e-benchmarking";
+# LATEST_TAG=$(curl -s "https://api.github.com/repos/cloud-bulldozer/e2e-benchmarking/releases/latest" | jq -r '.tag_name');
+# TAG_OPTION="--branch $(if [ "$E2E_VERSION" == "default" ]; then echo "$LATEST_TAG"; else echo "$E2E_VERSION"; fi)";
+# git clone $REPO_URL $TAG_OPTION --depth 1
+# pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+# export WORKLOAD=olm
+
+#   echo "Indexing Management cluster stats"
+#   METADATA=$(cat << EOF
+# {
+# "uuid": "${UUID}",
+# "workload": "${WORKLOAD}",
+# "mgmtClusterName": "${MC_NAME}",
+# "hostedClusterName": "${HC_NAME}",
+# "timestamp": "$(date +%s%3N)"
+# }
+# EOF
+# )
+
+if [[ "${ENABLE_LOCAL_INDEX}" == "true" ]]; then
+    EXTRA_FLAGS+=" --local-indexing"
+fi
+EXTRA_FLAGS+=" --gc-metrics=true --profile-type=${PROFILE_TYPE}"
+
+cmd="${KUBE_DIR}/kube-burner-ocp ${WORKLOAD} --log-level=${LOG_LEVEL} --qps=${QPS} --burst=${BURST} --gc=${GC} --uuid ${UUID} --iterations=${ITERATIONS} --metrics-endpoint=${METRICS_ENDPOINT}"
```

Comment:
The run.sh script in e2e-benchmarking already takes care of download the right version of kube-burner